desktop/ui/src/main/java/org/datacleaner/windows/FileTransferProgressWindow.java

```diff
@@ -32,7 +32,6 @@
 import org.apache.commons.lang.ArrayUtils;
 import org.apache.metamodel.util.Action;
 import org.datacleaner.actions.DownloadFilesActionListener;
-import org.datacleaner.actions.PublishResultToMonitorActionListener;
```

Comment:
There's a reference to this class in the JavaDoc below

Reply:
Fixed